### ISSUE TITLE
[Feature Request]: Module: enhancedTransportRequests

### ISSUE THREAD

**Issue by jxn-30** [LSS-Manager/LSSM-V.4#35]
**Is your feature request related to a problem?**
processing transport requests is an annoying work. An enhancement should ease the process.

**Solution / Feature**
Features:
- [ ] Available in Mission window
- [ ] Available in transport window
- [ ] Filter in the LSSM Settings (switchable) 
- [ ] Filter & sort by
    - [ ] distance
    - [ ] amount of free beds / cells
    - [ ] Patients only: Has matching hospital extension
    - [ ] Tax
        - [ ] First 5 over tax also
    - [ ] enough free beds / cells for mass-transport
    - [ ] text string
- [x] if possible close lightbox after last S5
- [ ] merge own and alliance hospitals / cell buildings
- [x] allow automatic click on "Next speech request"
- [x] open first transport request in mission window automatically

**Alternatives**
––– no alternatives –––

**Additional**
––– currently no additional notes –––

**Comment by JRH-1997** [comment]
Option to create one list of own and alliance buildings, so you can select the nearest hospital/cell that's withing the filter options.

**Comment by jxn-30** [comment]
Added as
> - [ ] merge own and alliance hospitals / cell buildings

**Comment by Cr4zyc4k3** [comment]
Isnt this already finished in the redesign module?

**Comment by jxn-30** [comment]
yes, but I would also like to provide it as an own module as I don't want to force users to use redesign :)

**Comment by ACH1980** [comment]
A button to close all transport request without transport would be nice.
